app/src/main/java/com/android/partagix/ui/screens/LoginScreen.kt

```diff
@@ -54,87 +69,197 @@ fun LoginScreen(authentication: Authentication, modifier: Modifier = Modifier) {
-                )
-                Text(
-                    text = "Sign in with Google",
-                    color = Color.Black,
-                    textAlign = TextAlign.Center,
-                    style = TextStyle(fontSize = 15.sp, fontWeight = FontWeight.Medium),
-                    modifier =
-                        modifier
-                            .wrapContentHeight(align = Alignment.CenterVertically)
-                            .padding(6.dp))
-              }
+/**
+ * Custom composable to display a grid of images blurred at the end.
+ *
+ * @param modifier Modifier to apply to this layout.
+ * @param images List of image resources to display.
+ * @param onShowSignInChange Callback to change the visibility of the sign in screen.
+ */
+@Composable
+fun ImageGrid(
+    modifier: Modifier = Modifier,
+    images: List<Int>,
+    onShowSignInChange: (Boolean) -> Unit
+) {
+  Box(modifier = modifier) {
+    LazyVerticalGrid(
+        columns = GridCells.Fixed(2),
+        contentPadding = PaddingValues(20.dp),
+        verticalArrangement = Arrangement.spacedBy(10.dp),
+        horizontalArrangement = Arrangement.spacedBy(10.dp),
+        userScrollEnabled = true,
+        modifier = Modifier.fillMaxWidth().testTag("ImageGrid")) {
+          items(images.size) { index ->
+            Surface(
+                shape = RoundedCornerShape(8.dp),
+                border = BorderStroke(2.dp, MaterialTheme.colorScheme.secondary),
+                modifier = Modifier.aspectRatio(1f)) {
+                  Image(
+                      painter = painterResource(id = images[index]),
+                      contentDescription = null,
+                      contentScale = ContentScale.Crop,
+                      modifier = Modifier.fillMaxSize())
+                }
+          }
         }
-      }
-    }
+    Column(
+        modifier = Modifier.fillMaxWidth().align(Alignment.BottomCenter).testTag("BlurEffectBox"),
+        verticalArrangement = Arrangement.Bottom) {
+          Box(
+              modifier =
+                  Modifier.fillMaxWidth()
+                      .height(170.dp)
+                      .align(Alignment.CenterHorizontally)
+                      .background(
+                          Brush.verticalGradient(
+                              colors =
+                                  listOf(Color.Transparent, MaterialTheme.colorScheme.background)))
+                      .blur(
+                          radius = 10.dp,
+                      ))
+          Box(
+              modifier =
+                  Modifier.fillMaxWidth()
+                      .height(120.dp)
+                      .background(color = MaterialTheme.colorScheme.background))
+        }
+    ClickableText(
+        modifier =
+            Modifier.align(Alignment.BottomCenter)
+                .padding(bottom = 70.dp)
+                .testTag("SeeMoreClickableText"),
+        text = AnnotatedString(text = "See 99+ more"),
```

Comment:
Same for here, do we really want static text ?

Reply:
Also the text seems to be black when dark mode is enabled, which is hard to read. The MaterialTheme is missing

Reply:
You want the text to by dynamic ? Except for language I don't really see the point, can you explain ?
Good catch for the materialTheme

Reply:
I guess it's about the number. But same argument as before, before we have real data in the db it's better like this.

Reply:
I added a theme color and checked in dark mode. It seems good now.

Reply:
Nice fix